app/preflight.mjs

```diff
@@ -0,0 +1,13 @@
+export default function Preflight({ req }) {
+  return {
+    pageTitle: getPageTitle(req.path)
+  }
+}
+
+function getPageTitle(path) {
+  const titleMap = {
+    //'/': 'Home'
+  }
+
+  return titleMap[path] || 'Enhance Starter Project'
```

Comment:
So here, I'm wondering if we should actually use the `titleMap` to drive the usage home a little more. And then maybe even not set a fallback, so that folks might more quickly catch an `undefined` in their page titles that aren't accounted for…?

```suggestion
function getPageTitle(path) {
  const titleMap = {
    '/': 'Enhance Starter Project'
  }

  return titleMap[path]
```